data_adapter_oemof/build_datapackage.py

```diff
@@ -91,56 +97,193 @@ def __split_timeseries_into_years(parametrized_sequences):
+                    new_foreign_keys.append(
+                        {
+                            "fields": field.name,
+                            "reference": {
+                                "resource": f"{mapper.process_name}_sequence"
+                            },
+                        }
+                    )
+                elif any(components[field.name].isin(mapper.timeseries.columns)):
+                    warnings.warn(
+                        "Not all profile columns are set within the given profiles."
+                        f" Please check if there is a timeseries for every Component in {mapper.process_name}"
+                    )
+                    new_foreign_keys.append(
+                        {
+                            "fields": field.name,
+                            "reference": {
+                                "resource": f"{mapper.process_name}_sequence"
+                            },
+                        }
+                    )
+                else:
+                    # Most likely the field may be a Timeseries in this case, but it is a scalar or unused.
+                    pass
+        return new_foreign_keys
+
+    def save_datapackage_to_csv(self, destination: str) -> None:
         """
         Saving the datapackage to a given destination in oemof.tabular readable format
-        :param destination:
-        :return:
+
+        Parameters
+        ----------
+        self: DataPackage
+            DataPackage to save
+        destination: str
+            String to where the datapackage save to. More convenient to use os.path. If last level of folder stucture
+            does not exist, it will be created (as well as /elements and /sequences)
+
+        Returns
+        -------
+        None if the Datapackage has been saved correctly (no checks implemented)
+
         """
+        # Check if filestructure is existent. Create folders if not:
+        elements_path = os.path.join(destination, "data", "elements")
+        sequences_path = os.path.join(destination, "data", "sequences")
+
+        os.makedirs(elements_path, exist_ok=True)
+        os.makedirs(sequences_path, exist_ok=True)
+
+        # Save elements to elements folder named by keys + .csv
+        for process_name, process_adapted_data in self.parametrized_elements.items():
+            process_adapted_data.to_csv(
+                os.path.join(elements_path, f"{process_name}.csv"),
+                index=False,
+                sep=";",
+            )
+
+        # Save Sequences to sequence folder named as keys + _sequence.csv
+        for process_name, process_adapted_data in self.parametrized_sequences.items():
+            process_adapted_data.to_csv(
+                os.path.join(sequences_path, f"{process_name}_sequence.csv"),
+                sep=";",
+                index_label="timeindex",
+                date_format="%Y-%m-%dT%H:%M:%SZ",
+            )
+
```

Comment:
We still miss to write the information about the multi-period into  `periods.csv`. We will do this in a new PR